packages/turf-boolean-intersects/README.md

```diff
@@ -12,7 +12,7 @@ Boolean-intersects returns (TRUE) if the intersection of the two geometries is N
 *   `feature2` **([Geometry][1] | [Feature][2]\<any>)** GeoJSON Feature or Geometry
 *   `options` **[Object][3]** Optional parameters (optional, default `{}`)
 
-    *   `options.ignoreSelfIntersections` **[boolean][4]** ignores self-intersections on input features (optional, default `false`)
```

Comment:
change not related to PR - but it seems that the docs were not properly updated previously.